Cabal/Distribution/Simple/Build/Macros/Z.hs

```diff
@@ -1,6 +1,12 @@
-{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE DeriveGeneric     #-}
```

Comment:
This file is autogenerated. I'll add a header to it asap. See https://hackage.haskell.org/package/zinza

This is why CI *Quick jobs* fails.